crud.py

```diff
@@ -0,0 +1,62 @@
+from sqlalchemy.orm import Session
+
+import models
+import schemas
+
+
+def get_all_authors(db: Session, skip: int = 0, limit: int = 10):
```

Comment:
if you use annotations, use them for return types as well

Reply:
Fixed